<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyQStackedWidget</class>
 <widget class="QWidget" name="MyQStackedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyQStackedWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnPage1">
       <property name="text">
        <string>第一页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPage2">
       <property name="text">
        <string>第二页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPage3">
       <property name="text">
        <string>第三页（代码添加一页）</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>600</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; color:#333333; background-color:#ffffff;&quot;&gt;      Qt&lt;/span&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:12px; color:#3366cc; background-color:#ffffff;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; color:#333333; background-color:#ffffff;&quot;&gt;是一个1991年由Qt Company开发的&lt;/span&gt;&lt;a href=&quot;https://baike.baidu.com/item/%E8%B7%A8%E5%B9%B3%E5%8F%B0/8558902&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; text-decoration: underline; color:#136ec2; background-color:#ffffff;&quot;&gt;跨平台&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://baike.baidu.com/item/C%2B%2B/99272&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; text-decoration: underline; color:#136ec2; background-color:#ffffff;&quot;&gt;C++&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://baike.baidu.com/item/%E5%9B%BE%E5%BD%A2%E7%94%A8%E6%88%B7%E7%95%8C%E9%9D%A2/3352324&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; text-decoration: underline; color:#136ec2; background-color:#ffffff;&quot;&gt;图形用户界面&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://baike.baidu.com/item/%E5%BA%94%E7%94%A8%E7%A8%8B%E5%BA%8F/5985445&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; text-decoration: underline; color:#136ec2; background-color:#ffffff;&quot;&gt;应用程序&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; color:#333333; background-color:#ffffff;&quot;&gt;开发框架。它既可以开发GUI程序，也可用于开发非GUI程序，比如&lt;/span&gt;&lt;a href=&quot;https://baike.baidu.com/item/%E6%8E%A7%E5%88%B6%E5%8F%B0/2438626&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; text-decoration: underline; color:#136ec2; background-color:#ffffff;&quot;&gt;控制台&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; color:#333333; background-color:#ffffff;&quot;&gt;工具和&lt;/span&gt;&lt;a href=&quot;https://baike.baidu.com/item/%E6%9C%8D%E5%8A%A1%E5%99%A8/100571&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; text-decoration: underline; color:#136ec2; background-color:#ffffff;&quot;&gt;服务器&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; color:#333333; background-color:#ffffff;&quot;&gt;。Qt是&lt;/span&gt;&lt;a href=&quot;https://baike.baidu.com/item/%E9%9D%A2%E5%90%91%E5%AF%B9%E8%B1%A1/2262089&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; text-decoration: underline; color:#136ec2; background-color:#ffffff;&quot;&gt;面向对象&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; color:#333333; background-color:#ffffff;&quot;&gt;的框架，使用特殊的&lt;/span&gt;&lt;a href=&quot;https://baike.baidu.com/item/%E4%BB%A3%E7%A0%81/86048&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; text-decoration: underline; color:#136ec2; background-color:#ffffff;&quot;&gt;代码&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; color:#333333; background-color:#ffffff;&quot;&gt;生成扩展（称为元对象编译器(Meta Object Compiler, moc)）以及一些&lt;/span&gt;&lt;a href=&quot;https://baike.baidu.com/item/%E5%AE%8F/2648286&quot;&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; text-decoration: underline; color:#136ec2; background-color:#ffffff;&quot;&gt;宏&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Helvetica Neue,Helvetica,Arial,PingFang SC,Hiragino Sans GB,Microsoft YaHei,WenQuanYi Micro Hei,sans-serif'; font-size:14px; color:#333333; background-color:#ffffff;&quot;&gt;，Qt很容易扩展，并且允许真正地组件编程。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="MyApplication.qrc">:/image/image/mascot.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyApplication.qrc"/>
 </resources>
 <connections/>
</ui>
